ci/kokoro/docker/build.sh

```diff
@@ -28,6 +28,10 @@ export CMAKE_SOURCE_DIR="."
 
 export GOOGLE_CLOUD_CPP_GENERATOR_RUN_INTEGRATION_TESTS="yes"
 
+# Define the build tool so other scripts (e.g. cache download),
+# currently the possible values are `CMake` or `Bazel`.
+export BUILD_TOOL="CMake"
```

Comment:
NO CHANGE NECESSARY: I wonder if in a future PR we could call the tarball-cache functions from the in-docker build scripts, which would eliminate the need to specify the BUILD_TOOL separately the existing `in_docker_script`.

Reply:
Yeah, I have been wondering something like that myself.  We would need to figure out how to pass the credentials file to the in-docker script, and make sure `gsutil` is installed and configured.